segmentstore/contracts/src/main/java/io/pravega/segmentstore/contracts/StreamSegmentInformation.java

```diff
@@ -76,21 +91,21 @@ private StreamSegmentInformation(String name, long startOffset, long length, boo
      */
     public static StreamSegmentInformationBuilder from(SegmentProperties base) {
         return StreamSegmentInformation.builder()
-                                       .name(base.getName())
-                                       .startOffset(base.getStartOffset())
-                                       .length(base.getLength())
-                                       .sealed(base.isSealed())
-                                       .deleted(base.isDeleted())
-                                       .lastModified(base.getLastModified())
-                                       .attributes(base.getAttributes());
+                .name(base.getName())
```

Comment:
Is this change necessary? The previous indentation seemed ok.

Reply:
this change is harmless too.

Reply:
reverted white space change.